packages/dds/cell/README.md

```diff
@@ -1,3 +1,43 @@
 # @fluidframework/cell
 
-Documentation available at https://fluidframework.com/docs/apis/cell/.
+The `SharedCell` Distributed Data Structure (DDS)  stores a single, shared value that can be edited or deleted.
+
+<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE) -->
```

Comment:
Curiosity: how do we auto-generate this file?

Reply:
It's done using the `markdown-magic` infra we have under `docs`.

https://github.com/microsoft/FluidFramework/tree/main/docs#markdown-magic

Right now it is generated by running the build in the docs directory, which isn't ideal. Longer term we want to have each package do its own docs generation independently, but that will likely have to wait until after we've finished our migration to pnpm/yarn.